src/posts/xarray-kvikio/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Enabling GPU-native analytics with Xarray and kvikIO"
+date: "2022-08-25"
+authors:
+  - name: Deepak Cherian, Wei Ji Leong
+    github: dcherian, weiji14
+summary: "An experiment with direct-to-GPU reads from a Zarr store using Xarray."
+---
+
+## TLDR
+
+We [demonstrate](https://github.com/xarray-contrib/cupy-xarray/pull/10) registering an Xarray backend that reads data from a Zarr store directly to GPU memory as [CuPy arrays](https://cupy.dev) using the new [kvikIO library](https://docs.rapids.ai/api/kvikio/stable/) and [GPU Direct Storage](https://developer.nvidia.com/blog/gpudirect-storage/) technology. This allows direct-to-GPU reads and GPU-native analytics on existing pipelines 🎉 😱 🤯 🥳.
+
+## Background
+
+### What is GPU Direct Storage?
+
+Quoting [this nVIDIA blogpost](https://developer.nvidia.com/blog/gpudirect-storage/)
+
+> I/O, the process of loading data from storage to GPUs for processing, has historically been controlled by the CPU. As computation shifts from slower CPUs to faster GPUs, I/O becomes more of a bottleneck to overall application performance. Just as GPUDirect RDMA (Remote Direct Memory Address) improved bandwidth and latency when moving data directly between a network interface card (NIC) and GPU memory, a new technology called GPUDirect Storage enables a direct data path between local or remote storage, like NVMe or NVMe over Fabric (NVMe-oF), and GPU memory. Both GPUDirect RDMA and GPUDirect Storage avoid extra copies through a bounce buffer in the CPU’s memory and enable a direct memory access (DMA) engine near the NIC or storage to move data on a direct path into or out of GPU memory, all without burdening the CPU or GPU. For GPUDirect Storage, storage location doesn’t matter; it could be inside an enclosure, within the rack, or connected over the network.
```

Comment:
@weiji14 this didn;'t reneder correctly with newlines.

Also thanks for the repeated reviews, I added you as co-author (I hope the syntax works)